src/components/AsideHeader/__stories__/AsideHeader.stories.tsx

```diff
@@ -9,5 +9,12 @@ export default {
     component: AsideHeader,
 } as Meta;
 
-const ShowcaseTemplate: Story = () => <AsideHeaderShowcase />;
+const ShowcaseTemplate: Story = (args) => <AsideHeaderShowcase {...args} />;
 export const Showcase = ShowcaseTemplate.bind({});
+
+const MultipleTooltipTemplate: Story = (args) => <AsideHeaderShowcase {...args} />;
```

Comment:
I think that a separate story is not needed for multiple tooltips, they can be seen in the main showcase

Reply:
What about 3 stories: Showcase, Compact, MultipleTooltip? I think it would be nice for user to see all cases

Reply:
Check last commit

Reply:
Only multiple tooltips should be enabled by default. Then the third story will be about single tooltips